example_scripts/download_vhi_parallel.py

```diff
@@ -138,8 +150,13 @@ def test(parallel=True, pool=None):
         # ipdb.set_trace()
     return
 
+
 if __name__ == "__main__":
     # pool = multiprocessing.Pool(processes=100)
     pool = Pool(processes=100)
-    main(pool)
     # test(pool=pool)
+
+    # main(pool)
```

Comment:
commented this out which is so bad but just didn't want you downloading 100GB of data